workflow/applications/gfs_cycled.py

```diff
@@ -128,7 +128,7 @@ def _get_app_configs(self):
     @staticmethod
     def _update_base(base_in):
 
-        return GFSCycledAppConfig.get_gfs_cyc_dates(base_in)
+        return base_in
```

Comment:
Is this function needed anymore? Doesn't look like it's doing anything anymore and just returns what goes into it.

Reply:
It gets called by the superclass, so it does need to be defined.

Reply:
Ah, thanks for explaining @DavidHuber-NOAA !